source/_integrations/tuya.markdown

```diff
@@ -26,7 +26,7 @@ ha_dhcp: true
 
 The Tuya Home Assistant integration is developed for controlling **Powered by Tuya (PBT)** devices, which is officially maintained by the Tuya Developer Team.
 
-You can find more information for this integration on [Tuya Integration Github](https://github.com/tuya/tuya-home-assistant).
+You can find more information for this integration on [Tuya Integration Github](https://github.com/tuya/tuya-home-assistant?_source=418498f8d2819b5b1654a0c12c5325cd).
```

Comment:
This is the documentation for the core integration, we should not refer to the external integration in the core documentation.

```suggestion
```

Reply:
fixed.